### ISSUE TITLE
[cli] support core 3.X

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/patternlab-node#813]

**Comment by bmuenzenmeyer** [comment]
@raphaelokon your branch `feature/cli-core-align` looked pretty good to me. shall we merge it into `dev` and keep polishing, or are you still working through anything?

**Comment by raphaelokon** [comment]
@bmuenzenmeyer Nope. I think it is good to go. I have some open tickets over in the old repo, but I'll move those here. Also: I need to revise some flags (like the watch flag) as it is now obsolete since CLI proxies to core :-)

**Comment by bmuenzenmeyer** [comment]
@raphaelokon how is this coming? I havent tested CLI with 3.X yet. Is it ready>

**Comment by raphaelokon** [comment]
The only missing bit is global CLI installs. Haven't tested that.
